### ISSUE TITLE
Strange behaviour when grid has a defined set of rows and is completely filled up

### ISSUE THREAD

**Issue by DevHachi** [gridstack/gridstack.js#1858]
First of all I want to thank you for all your effort you've put into this awesome library.
But currently I have come across a strange bug, maybe you could point me in the right direction.
Maybe I am doing something wrong ;)

Currently it seems whenever I have a fixed row or an max row set and the grid is completely filled, then it started to make some wrong assumption when I try to move them around inside the grid.

Here is an playground for you to test: https://jsfiddle.net/ChiangChu/8Lpx0kau/11/

You can reproduce it about 100% when drag an element outside the grid and then back inside. For some reason other elements that does not collide with the object start to move and you can even overlap two elements completely.

I would be really appreciate any hint, that could cause this error.

**Gridstack Version 4.2.7**

Best Regards

**Comment by adumesny** [comment]
yep, I see the issue - cleaned up a ig https://jsfiddle.net/adumesny/smbL4y60/

**Comment by adumesny** [comment]
likely a hard to fix issue (collision and dealing with dragging back from outside, in full grid)

**Comment by adumesny** [comment]
closing due to age
